src/main/java/org/geppetto/model/neuroml/modelInterpreterUtils/PopulateContinuousProjectionTypes.java

```diff
@@ -77,10 +78,10 @@ public Type resolveProjectionImportType(Component projection, ImportType importT
 	 * @throws ModelInterpreterException
 	 */
 	private Variable extractConnection(Component projectionChild, ArrayType prePopulationType, Variable prePopulationVariable, ArrayType postPopulationType, Variable postPopulationVariable)
-			throws ModelInterpreterException
+            throws ModelInterpreterException, GeppettoVisitingException
 	{
 		ConnectionType connectionType = (ConnectionType) populateTypes.getTypeFactory().getSuperType(ResourcesDomainType.CONNECTION);
-
+                connectionType.getSuperType().add(this.geppettoModelAccess.getType(TypesPackage.Literals.CONNECTION_TYPE));
```

Comment:
@tarelli I guess it's a symptom of some code duplication here, but I think it needs to be added in these 3 places as things stand, unless eg. 2 of these methods are never called